README.md

```diff
@@ -1,4 +1,7 @@
-# AmpliPro Home Audio System
+# AmpliPi Home Audio Software
```

Comment:
I think System is a better description. Software is overly generic and useless its like putting AI in the name of something.

One potential option if system is off the table is Platform but I still prefer System here.

Reply:
I changed it to system. I think you're right.